relationships/synthesis/EXT_INFRA-KUBERNETES_CONTAINER-to-EXT-SERVICE.yml

```
@@ -0,0 +1,36 @@
+relationships:
+  - name: extInfraK8sContainerEXTService
```

I don't understand why this and the file name are prefixed with ext.

```suggestion
  - name: infraK8sContainerEXTService
```

---

This infra data is coming from the opentelemetry(external service) not from newrelic infra agent so to differentiate I have added ext.

---

@naxhh Here we have 2 cases.
Case 1: Configure infra using opentelemetry(Infra metrics are coming from external source).
Case 2: Configure infra using the NR infra agent.
This relationship is for case 1(`extInfraK8sContainerEXTService`) and I will create a separate relationship synthesis for case 2(`nrInfraK8sContainerEXTService`).